pro/src/components/AddressManual/AddressManual.tsx

```diff
@@ -0,0 +1,93 @@
+import { parseDms } from 'dms-conversion'
+import { useField, useFormikContext } from 'formik'
+import { useEffect } from 'react'
+
+import { Callout } from 'components/Callout/Callout'
+import { CalloutVariant } from 'components/Callout/types'
+import { FormLayout } from 'components/FormLayout/FormLayout'
+import fullLinkIcon from 'icons/full-link.svg'
+import { ButtonLink } from 'ui-kit/Button/ButtonLink'
+import { ButtonVariant } from 'ui-kit/Button/types'
+import { TextInput } from 'ui-kit/form/TextInput/TextInput'
+import { getCoordsType } from 'utils/coords'
+
+import styles from './AddressManual.module.scss'
+
+export const AddressManual = (): JSX.Element => {
+  const [coords, coordsMeta] = useField('coords')
+
+  const { setFieldValue } = useFormikContext()
+
+  useEffect(() => {
+    if (coords.value && !coordsMeta.error) {
+      const coordsType = getCoordsType(coords.value)
+
+      if (coordsType === 'DD') {
+        const [latitude, longitude] = coords.value.split(', ')
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('latitude', latitude)
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('longitude', longitude)
+      } else if (coordsType === 'DMS') {
+        const [latitude, longitude] = coords.value.split(' ')
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('latitude', parseDms(latitude))
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('longitude', parseDms(longitude))
+      } else {
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('latitude', '')
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('longitude', '')
+      }
+    }
```

Comment:
(proposition/avis perso) j'avoue qu'avec tout les commentaires, ça me parait complexe, un truc du genre me semble un peu plus lisible

```suggestion
      const coordsType = getCoordsType(coords.value)
      let lat, lng
      if (coordsType === 'DD') {
        ...
      } else if (coordsType === 'DMS') {
      ...
        parse
      } 
      setFieldValue...
    }
```

Reply:
Done